clifford/numba/_multivector.py

```diff
@@ -64,7 +63,7 @@ def _numba_type_(self):
     except KeyError:
         # Computing and hashing `dtype_type` is slow, so we do not use it as a
         # hash key. The raw numpy dtype is much faster to use as a key.
-        dtype_type = _numpy_support.from_dtype(dt)
+        dtype_type = _numpy_support.from_dtype(dt)[::1]  # c-order
         ret = cache[dt] = MultiVectorType(layout_type, dtype_type)
```

Comment:
These are now misnamed:
```suggestion
        value_type = _numpy_support.from_dtype(dt)[::1]  # c-order
        ret = cache[dt] = MultiVectorType(layout_type, value_type)
```

Reply:
And I suppose the comment above